packages/expo-google-sign-in/src/index.js

```diff
@@ -0,0 +1,136 @@
+// @flow
+import { NativeModulesProxy } from 'expo-core';
+
+import User from './User';
+
+const { ExpoGoogleSignIn } = NativeModulesProxy;
+
+export const { ERRORS, SCOPES, TYPES } = ExpoGoogleSignIn;
+
+export type GoogleSignInType = TYPES.DEFAULT | TYPES.GAMES;
+
+export type GoogleSignInOptions = {
+  scopes: ?Array<string>,
+  webClientId: ?string,
+  hostedDomain: ?string,
+  accountName: ?string,
+
+  // Android
+  signInType: ?GoogleSignInType,
+  isOfflineEnabled: ?boolean,
+  isPromptEnabled: ?boolean,
+  // iOS
+  clientId: ?string,
+  language: ?string,
+  openIdRealm: ?string,
```

Comment:
it seems weird to me that all of these things are platform-specific. not sure if there's anything to do about it other than make sure to document them. thx google 🤷‍♂️